ui/app/abilities/variable.js

```diff
@@ -38,10 +39,48 @@ export default class Variable extends AbstractAbility {
   )
   canDestroy;
 
-  @computed('rulesForNamespace.@each.capabilities')
-  get policiesSupportVariableView() {
-    return this.rulesForNamespace.some((rules) => {
-      return get(rules, 'SecureVariables');
+  @computed('token.selfTokenPolicies')
+  get policiesSupportVariableList() {
+    return this.policyNamespacesIncludeSecureVariablesCapabilities(
+      this.token.selfTokenPolicies,
+      ['list', 'read', 'write', 'destroy']
+    );
```

Comment:
`nit` `suggestion`: The first parameter is guaranteed to always be this value and the second parameter is a hard coded array. The function API for `policyNamespacesIncludeSecureVariablesCapabilities` should just be a computed getter that returns a boolean instead.

Reply:
A getter won't let us pass parameters; there are plenty of other situations where we would want a generic "check to see if my policy support this thing, generally" for variables — the `can create` capability could leverage this to determine if it should even bother showing the "Create New Variable" button by passing in only `['write']` for example.

Reply:
path is a bound property of this ability and accessible via `this.path` just like namespace.